core/services/feeds/proto/feeds_manager.pb.go

```diff
@@ -1,17 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.26.0
```

Comment:
probably best not to downgrade the version

Reply:
This matches the version that FMS uses. I think it changed because of the last time some one updated the proto file and they generated with a higher version. We can look to standardise the version to the latest in a separate commit, but for now will leave as is so it matches the FMS version